frontend/src/metabase/ui/components/inputs/Checkbox/index.ts

```diff
@@ -0,0 +1 @@
+export { Checkbox } from "@mantine/core";
```

Comment:
We'll need to set a default prop for the checkbox to use our icon component somehow. Any ideas on the best pattern for that? https://mantine.dev/core/checkbox/#replace-icon has some related info.

Reply:
Ok sounds good - I wasn't sure if we were going to use our own icons for the Mantine stuff just yet, the default icon makes me super uncomfortable 😅